autospec/specfiles.py

```diff
@@ -641,22 +641,39 @@ def write_maven_install(self):
         """Write installation steps to specfile for maven source packages."""
         self.write_license_files()
 
+        # Iterate over all target subdirs
+        self._write_strip("for targetdir in `find . -type d -name target`; do")
```

Comment:
For consistency in shell code style, I would prefer seeing `$(...)` here rather than the backticks.

Reply:
Agreed.